judgments/urls.py

```diff
@@ -38,22 +38,10 @@
         name="feed",
     ),
     re_path(
-        r"^(?P<document_uri>.*/\d{4}/\d+.*)/data.pdf$",
-        get_best_pdf,
-        name="detail_pdf",
+        r"^(?P<document_uri>.*/\d{4}/\d+.*)/?$",
```

Comment:
I'm struggling to see how this works at the moment

Reply:
I've made some tweaks to the urls and the resolver to make it easier to grok now